src/app/cup/cup-rating-group-table/cup-rating-group-table.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
```

Comment:
please move this component to `src/app/cup/shared` folder. All non-page components should be there